docs/reference/signer-config.md

```diff
@@ -1,60 +1,100 @@
 <!--[metadata]>
 +++
-title = "Signer Configuration File"
-description = "Specifies the configuration file for Notary Signer"
-keywords = ["docker, notary, notary-signer, configuration"]
+title = "Signer Configuration"
+description = "Configuring the notary client, server and signer."
+keywords = ["docker, notary, notary-client, notary-server, notary server, notary-signer, notary signer"]
 [menu.main]
-parent="mn_notary"
-weight=90
+parent="mn_notary_config"
 +++
 <![end-metadata]-->
 
-# Notary Signer Configuration File
+
+# Notary signer configuration file
+
+This document is for power users of the [notary client](../advanced_usage.md),
+or for those who are [running their own service](../running_a_service.md) who
+want to facilitate CLI interaction or specify custom options.
+
+## Overview
+
+Notary signer [requires environment
+variables](#environment-variables-required-if-using-mysql) to encrypt private
+keys at rest. It also requires a configuration file, the path to which is
+specified on the command line using the `-config` flag.
+
+A Notary signer configuration file consists of the following sections:
+
+<table>
+	<tr>
+		<td><a href="#server-section-required">server</a></td>
```

Comment:
Also apologies :|  these dashes in the section links also may have to be changed to underscores.